EIPS/eip-1.md

```diff
@@ -3,121 +3,108 @@ eip: 1
 title: EIP Purpose and Guidelines
 status: Active
 type: Meta
-author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>
+author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, William Entriken <github.com@phor.net>
```

Comment:
What is the process for adding an `author` to an EIP?  In particular, it feels like we should be conservative with authors on EIP-1, especially with the auto-accept bot.

Reply:
EIP-1 isn't draft, so the auto-accept bot won't automerge any changes to it.

Reply:
@MicahZoltu My name is no longer there, now it is a link to all authors.

Does this resolve your concern? If yes, will you be able to turn your comment into an "Approve" vote?

Reply:
I'm not an editor, my approval holds to value. 😄